bin/proving-service/src/commands/worker.rs

```diff
@@ -65,9 +62,12 @@ pub struct StartWorker {
     /// The port of the worker
     #[clap(long, default_value = "50051")]
     port: u16,
+    /// The port of the HTTP server for the information endpoint
+    #[clap(short, long, default_value = "3000")]
```

Comment:
Should we remove the `short` argument in case it results in a conflict later on (and to make it consistent with other parameters)?

Reply:
This field isn't needed, i'm removing it.